scss/components/_case-list-table.scss

```diff
@@ -0,0 +1,139 @@
+.civicase__table--caselist {
+  .fixed-col {
+    .label {
+      font-size: 0.8em;
```

Comment:
Where is this style used?

Reply:
<img width="1598" alt="c51-38 - fixed col label" src="https://user-images.githubusercontent.com/3340537/44709916-77822880-aac8-11e8-87a2-d10c3c8d4287.png">
For label badges in the left col This will be removed as we further develop the the left case list card.